.vscode/launch.json.example

```diff
@@ -0,0 +1,24 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Debug Tests",
+            "type": "debugpy",
+            "request": "launch",
+            "program": "${file}",
+            "purpose": ["debug-test"],
+            "console": "integratedTerminal",
+            "justMyCode": false,
+
+            // This is needed because pytest-cov is incompatible with using the
+            // debugger. See https://code.visualstudio.com/docs/python/testing#_pytest-configuration-settings
+            "env": {"PYTEST_ADDOPTS": "--no-cov"},
+
+            // Adds some magic for Django templates.
+            "django": true
+          }
```

Comment:
This launch config isn't working for me for some reason - when I run it via the "Run and Debug" panel with a test file open, it gives `No module named 'openedx_learning'`. But I'm wondering why we even need this launch configuration? Because the "Debug Test" buttons on the "Testing" panel are working just fine for me already.

Reply:
Interesting. I've always been running it from the "Testing" panel where it works. But I see the same issues as you in the Run/Debug panel. I'll look into that, thank you.

Reply:
Ah, I didn't think it was affecting the "Testing" panel but now I can see that when I have this code in place it runs the debug mode tests differently. For me the "debug test" button works fine either with or without this `launch.json` config. The only difference I can see is that with the config in place, it produces more output in the terminal.

I don't think it's an issue that it doesn't also work from "Run and Debug", as long as other people aren't confused like I was.